res/controllers/Traktor Kontrol S2 MK3.hid.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MixxxControllerPreset mixxxVersion="2.2.0" schemaVersion="1">
+  <info>
+    <name>Traktor Kontrol S2 MK3</name>
+    <author>Michael Schmidt</author>
+    <description>HID Mapping for Traktor Kontrol S2 MK3</description>
+    <wiki>https://www.mixxx.org/wiki/doku.php/native_instruments_traktor_s2_mk3</wiki>
```

Comment:
Please add the forum post URL, too.

Reply:
Done in https://github.com/mi01/mixxx/commit/d5822606d88dbaad3ff81d4a64d15c12f2ebfb95